Makefile

```diff
@@ -8,6 +8,7 @@ LOCAL_BOOTSTRAP_CONFIG = localenvoyconfig.yaml
 SECURE_LOCAL_BOOTSTRAP_CONFIG = securelocalenvoyconfig.yaml
 PHONY = gencerts
 ENVOY_IMAGE = docker.io/envoyproxy/envoy:v1.18.3
+GATEWAY_API_VERSION = $(grep "sigs.k8s.io/gateway-api" go.mod | awk '{print $2}')
```

Comment:
this isn't actually working, I get an empty string when i make the kubectl command below echo

```
kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=" -o examples/gateway/00-crds.yaml
```